web/ui/react-app/src/pages/Flags.test.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { mount, shallow, ReactWrapper } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import Flags from './Flags';
+import { Alert, Table } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+
+const sampleAlertmanagerResponse: {
```

Comment:
`sampleAlertmanagerResponse` -> `sampleFlagsResponse` (it has nothing to do with Alertmanager, I think you probably accidentally copied this from the example in another comment).

Reply:
... I think I was talking about alertmanager with hooten while writing this and got my thoughts crossed. Ooops.